nni/common/framework.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import importlib
+import os
+import sys
+from typing import Optional, cast
+
+from nni.typehint import Literal
+
+__all__ = ['set_default_framework', 'get_default_framework', 'shortcut_module', 'shortcut_framework']
```

Comment:
PEP 8 suggest to place this before `import`.

Reply:
Yeah... really counter-intuitive.